tests/test_utils.py

```diff
@@ -20,3 +21,9 @@ class TestCase(unittest.TestCase):
         [torch.zeros(1, 3, 3, 3), torch.ones(1, 3, 3, 3), torch.full((1, 3, 3, 3), 2)],
         1,
     )
+
```

Comment:
config as class variable might not support multiple different configs. 

Maybe using instance variables would be safer